style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #262626;
+}
+
+.cup {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  transform: translate(-50%, -50%);
+  width: 150px;
+  height: 180px;
+
+  border: 6px solid #262626;
+  border-top: 2px solid transparent;
+  border-radius: 15px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+
+  background: url(../images/wave2.png);
```

Comment:
just change :
background: url(../images/wave2.png); from to
***background: url(./images/wave2.png);***
your image folder path didn't find by css because you add extra dot there

Reply:
now your project will start run after this change